config/locales/en.yml

```diff
@@ -204,7 +204,7 @@ en:
       subject: 'Subject'
       subject_hint: 'e.g. Maths'
       job_title: 'Job title'
-      job_title_hint: 'e.g. Principal'
+      job_title_hint: 'e.g. Leader'
```

Comment:
Do we know this is a term teachers commonly use?

Reply:
Lots of jobs seem to use it 🤷‍♀️. Most of the searches seem to be for "teacher" which I'd like to discourage.